utils/discounts/registration.ts

```diff
@@ -0,0 +1,24 @@
+// Discount details to fill in case of a duplicate of this page
+export type RegistrationDiscount = {
+  upsell: Discount;
+  expiry: number;
+  priceDuration: number;
+  discountMailGroupId: string;
+};
+
+export const registrationDiscount: RegistrationDiscount = {
```

Comment:
`registrationDiscount` is too vague and could define every discount concerning registration. 

I would propose `registrationUpsell`, for renewal we could have `renewalUpsell`

As it'll be in the evergreen.ts we would know it's the one we use in an evergeen way.